src/pages/CreatorCenter/components/ClassRegist/components/InputForm.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import theme from '../../../../../styles/theme';
+
+function InputForm({ type, subject, description, borderUnUse }) {
+  return (
+    <Container borderUnUse={borderUnUse}>
+      <Subject>{subject}</Subject>
+
+      {type === 'list' ? (
+        <Select name="levelList">
+          <option value="">선택안함</option>
+          <option value="입문자">입문자</option>
+          <option value="초급자">초급자</option>
+          <option value="중급자">중급자</option>
+          <option value="준전문가">준전문가</option>
+          <option value="전문가">전문가</option>
+        </Select>
+      ) : type === 'textarea' ? (
+        <Textarea />
+      ) : (
+        <Input type={type} placeholder={description} />
+      )}
+    </Container>
```

Comment:
```suggestion
    <Container borderUnUse={borderUnUse}>
      <Subject>{subject}</Subject>
      {INPUTS[type] || <Input type={type} placeholder={description} />}
    </Container>
```

컴포넌트 내에서 조건에 따라 삼항연산자를 작성하는 방법으로 컴포넌트 재사용을 하신 부분은 너무나 좋지만!
삼항연산자를 중첩해서 사용할 경우 오히려 코드를 파악하는데에 어려움이 생길 수 있습니다.

제가 제안드리는 방법은 Select와 Textarea를 INPUT이라는 객체로 관리 한뒤
props로 받는 type에 따라서 맵핑하는 방식으로 구현하면 더욱 깔끔할 것 같네요!